src/sentry/tsdb/redissnuba.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import absolute_import
```

Comment:
This module name makes me eye twitch a little bit but I don't have any better ideas. 🤷‍♂️